README.md

```diff
@@ -14,7 +14,7 @@ This plugin requires Grunt `~0.4.2`
 If you haven't used [Grunt](http://gruntjs.com/) before, be sure to check out the [Getting Started](http://gruntjs.com/getting-started) guide, as it explains how to create a [Gruntfile](http://gruntjs.com/sample-gruntfile) as well as install and use Grunt plugins. Once you're familiar with that process, you may install this plugin with this command:
 
 ```shell
-npm install git://github.com/incuna/grunt-gettext-checker.git#0.1 --save-dev
+npm install git://github.com/incuna/grunt-gettext-checker.git#1.1.1 --save-dev
```

Comment:
Do you have to have the version number? You could have a `latest` tag that you update with every release, then you don't have to change the README every time.